recipe/meta.yaml

```
@@ -44,4 +44,5 @@ about:
 
 extra:
   recipe-maintainers:
+    - mikelambert
```

Can you also add `jenan-anthropic` as well? Thanks!
```suggestion
    - mikelambert
    - jenan-anthropic
```